packages/auth/src/authClients/firebase.ts

```diff
@@ -4,21 +4,34 @@ export type Firebase = typeof Firebase
 
 import { AuthClient } from './'
 
+// @TODO: Firebase doesn't export a list of providerIds they use
+// But I found them here: https://github.com/firebase/firebase-js-sdk/blob/a5768b0aa7d7ce732279931aa436e988c9f36487/packages/rules-unit-testing/src/api/index.ts
+export type oAuthProvider =
+  | 'google.com'
+  | 'facebook.com'
+  | 'github.com'
```

Comment:
It looks like you might be able to import those directly!

Reply:
I tried but isn't in the main exports I believe. Will take a look again!

Reply:
I would reach right in there!

Reply:
I see this is part of the JS sdk, which is not something that we're including as a dependency, maybe we don't need to do this... I like it either way 👍